source/mir/random/flex/internal/area.d

```diff
@@ -186,13 +190,13 @@ body
         }
         else if (iv.c == 1)
         {
-            area = S(0.5) * sh.a * intLength * (z + 2);
+            area = S(0.5) * sh.a * ivLength * (z + 2);
```

Comment:
Yea, matches what I understood.  I would suggest using `\tilde{f}` instead of `g` in your own writeup, just for clarity (I would define a custom LaTeX macro for it).